templates/webapps/galaxy/admin/center.mako

```diff
@@ -104,7 +104,7 @@ Please visit <a href="https://wiki.galaxyproject.org/Admin" target="_blank">the
         <h4>Sample tracking</h4>
             <ul>
                 <li>
-                    Please see the <a href="https://wiki.galaxyproject.org/Admin/DataLibraries/LibrarySampleTracking" target="_blank">sample tracking tutorial</a>.
+                    Please see the <a href="https://galaxyproject.org/admin/data-libraries/library-sample-tracking" target="_blank">sample tracking tutorial</a>.
```

Comment:
https://galaxyproject.org/archive/library-sample-tracking/

P.S.: https://wiki.galaxyproject.org/Admin/DataLibraries/LibrarySampleTracking is missing a proper redirect.